scripts/LedSignMock.py

```diff
@@ -0,0 +1,36 @@
+
+import curses
+import argparse
+
+parser = argparse.ArgumentParser(description="Prints mock output.")
+parser.add_argument('--mock-output', help = "Displays Mock Output", )
+args = parser.parse_args()
+
+menu = ['   ','SCE LED Sign Visualization ', ' ','Most Recent Health Check: ', 'Most Recent Sign Update: ',' ',
+'Current Sign Text:', 'Current Brightness:', 'Current Scroll Speed:', 'Current Text Color: ', 'Current Background:',
+'Current Border Color: ' ]
+
+class LedSignMock(data):
+    def print_menu(stdscr, selected_row_idx):
+        data = "info here"
+        stdscr.clear()
+        stdscr.border(0)
+        h, w = stdscr.getmaxyx()
+        for idx, row in enumerate(menu):
+            x = w//3 - len(menu)
+            y = h//3 - len(menu)//3 + idx
+            stdscr.addstr(y, x, row)
+        stdscr.refresh()
+
+    def main(stdscr):
```

Comment:
is `main` supposed to be a part of the class? it would be better to name it whatever it does like `print_sign_mock`